quad_control/scripts/quad_simulator.py

```diff
@@ -39,7 +40,12 @@
 from Simulators.AttitudeInnerLoop.SimulatorWithAttitudeInnerLoop import SimulatorWithAttitudeInnerLoop
 from Simulators.NoAttitudeInnerLoop.SimulatorWithNoAttitudeInnerLoop import SimulatorWithNoAttitudeInnerLoop
 
-simulators_dictionary = {0:SimulatorWithZeroDynamics,1:SimulatorWithAttitudeInnerLoop,2:SimulatorWithNoAttitudeInnerLoop}
```

Comment:
Ok, I wanted to have the dictionary in a separate file (like we do for the trajectories).
Then I can just use 0,1,2 as the keys and the GUI should work with the new dictionary too.